tests/utils/atomWithObservable.test.tsx

```diff
@@ -95,32 +70,3 @@ it('writable count state', async () => {
   fireEvent.click(getByText('button'))
   await findByText('count: 9')
 })
-
-// FIXME we would like to support retry
-it.skip('count state with error', async () => {
```

Comment:
This test wasn't valid - an observable can't recover from an error state. For the notion of "retrying" observables to work, the error would never reach `atomWithObservable`, so the retrying logic is out of scope for jotai.

Reply:
So, I didn't know about the observable error behavior, and created this test. But, it's an expected behavior.
Now, you find the actual issue of error async reads, and fix it. Very very nice.